### ISSUE TITLE
Better Closing Message Text: Need pre-close Liquidity and Chan Close TXN

### ISSUE THREAD

**Issue by zapomatic** [alexbosworth/ln-telegram#17]
I woke up this morning to find a telegram alert that Kollider had force-closed our channel (even though my node was online--but they are closing up shop messily and that's an aside).

<img width="469" alt="Screenshot 2024-01-28 at 9 02 05 AM" src="https://github.com/alexbosworth/ln-telegram/assets/155849471/e05dcc89-dbfa-4b50-ac83-4a3f78c3e8fa">

Two issues are apparent in the language of the telegram alerts:

1. Inbound liquidity now: `0.00000000` and Outbound liquidity now: `0.00000000`
2. The funding outpoint is shown but not the closing transaction

Since I had only recently opened the channel and thought the liquidity was all on my side, I did some quick math and with 8M sats starting point and `7,752,952` returned on chain, the telegram alerts make it look like I lost `251,486` sats on the force close. 
I had to dig around in my logs to find the channel close txn so I could see how much liquidity was on their side and calculate my cost for the closing transaction, which then turned out to be `10,907` -- much relief!

So it would be awesome if the telegram alerts could show:

1. The inbound/outbound at the time of close (not what it would be after close)
2. The closing transaction
3. If possible, calculate the final cost of the close transaction to our side

**Comment by alexbosworth** [comment]
I think it's a little tricky to know what things were at the time of close because LND doesn't have a historical view so by the time we see that there is a close happening it's too late to really know what the liquidity was before that event fired, but maybe there is a solution

The closing transaction is also tricky because we don't always see this from the API, and it's not really decided what the channel close is going to be just at closing time: there are multiple possible close scenarios: breach, force close local, force close remote, cooperative, all of these scenarios would result in different outcomes and none of them are known at the pending stage, only the closed stage (when a close confirms)

**Comment by zapomatic** [comment]
understood. I appreciate the quick eyes on it and thoughtful consideration. Something to noodle on as things evolve :)

**Comment by alexbosworth** [comment]
Yeah I think it could be improved though, it was probably made at a time when the API was less powerful and could do with another pass given new tooling to see what more can be shown
